core/src/com/fourinachamber/fortyfive/screen/general/ScreenBuilder.kt

```diff
@@ -522,7 +535,7 @@ class ScreenBuilder(val file: FileHandle) {
         val screenSchema: OnjSchema by lazy {
             OnjSchemaParser.parseFile(Gdx.files.internal("onjschemas/screen.onjschema").file())
         }
-
+        lateinit var curBuilder: ScreenBuilder
```

Comment:
this is not only ugly, it also doesn't work, because there can be more than one ScreenBuilder at a time

Reply:
This was unsused and i forgot to remove/delete it